mesonbuild/interpreter.py

```diff
@@ -2138,6 +2148,19 @@ def get_subproject_infos(self, kwargs):
             raise InterpreterException('Fallback info must have exactly two items.')
         return fbinfo
 
+    def check_version(self, name, dep, wanted, dirname, varname):
+        if dep is None:
+            return None
+
+        if wanted:
+            found = dep.version_method([], {})
+            if found == 'undefined' or not mesonlib.version_compare(found, wanted):
+                mlog.log('Subproject', mlog.bold(dirname), 'dependency',
+                        mlog.bold(varname), 'version is', mlog.bold(found),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->